bigbluebutton-html5/imports/startup/client/base.jsx

```diff
@@ -94,7 +94,7 @@ class Base extends Component {
     });
     Session.set('isFullscreen', false);
 
-    const users = Users.find({}, { fields: {
+    const users = Users.find({meetingId: Auth.meetingID}, { fields: {
```

Comment:
I think you can add `validated: true` and even `userId: $ne localUserId` - with the correct syntax

Reply:
👍